charactersheet/charactersheet/viewmodels/character/armor.js

```diff
@@ -15,13 +15,7 @@ function ArmorViewModel() {
         'armorName asc': { field: 'armorName', direction: 'asc'},
         'armorName desc': { field: 'armorName', direction: 'desc'},
         'armorType asc': { field: 'armorType', direction: 'asc'},
-        'armorType desc': { field: 'armorType', direction: 'desc'},
-        'armorDexBonus asc': { field: 'armorDexBonus', direction: 'asc'},
-        'armorDexBonus desc': { field: 'armorDexBonus', direction: 'desc'},
```

Comment:
That logic can be used for when we actually calculate AC bonuses. If you feel so inclined, feel free to delete it, but I don't think it's doing any harm there.